Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -133,7 +133,7 @@ private IEnumerable<string> GetObsoleteBranchNames(RefreshContext context, strin
             }
 
             bool withoutRegexFilter = string.IsNullOrEmpty(context.RegexFilter);
-            return _commandOutputParser.GetBranchNames(result.StandardOutput)
+            return _commandOutputParser.GetBranchNames(result.StandardOutput, context.IncludeRemotes)
```

Comment:
Should we rename "IncludeRemotes" to "RemoteBranches" for clarity?

Reply:
OK

Reply:
`RemoteBranches` sounds like a collection. Is `IsRemoteBranches` better?

Reply:
About the same to me, so keeping "RemoteBranches"